Build error - fatal: Invalid object name '@'.

In my centos compiling environment, this error happens when running make in lotus:
```
[root@localhost lotus]# make
make -C extern/go-bls-sigs/ libbls_signatures.a libbls_signatures.pc libbls_signatures.h
make[1]: Entering directory `/root/go/src/github.com/filecoin-project/lotus/extern/go-bls-sigs'
./install-bls-signatures
fatal: Invalid object name '@'.
acquiring release @ @:bls-signatures
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   129  100   129    0     0    102      0  0:00:01  0:00:01 --:--:--   102
jq: error (at <stdin>:4): Cannot iterate over null (null)
failed to download release (tag URL: https://api.github.com/repos/filecoin-project/bls-signatures/releases/tags/@:bls-signatures, response: {
  "message": "Not Found",
  "documentation_url": "https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name"
})
./install-bls-signatures: line 20: [: =: unary operator expected
build failed: could not obtain precompiled assets for bls-signatures - contact @laser or @dignifiedquire
make[1]: *** [.install-bls] Error 1
make[1]: Leaving directory `/root/go/src/github.com/filecoin-project/lotus/extern/go-bls-sigs'
make: *** [build/.bls-install] Error 2
```

The same error happened in go-filecoin early version, but it's gone in 0.5. One way to fix this is to change the related scripts, but I do not think it is the best way. Any one has better idea?

--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@ In order to run lotus, please do the following:
 1. Make sure you have these dependencies installed:
 - go (1.13 or higher)
 - gcc (7.4.0 or higher)
-- git
+- git (version 2 or higher)
 - bzr (some go dependency needs this)
 - jq
 - pkg-config

